spesialist-selve/src/main/kotlin/no/nav/helse/modell/utbetaling/LagreOppdragCommand.kt

```diff
@@ -121,13 +119,10 @@ internal class LagreOppdragCommand(
 
     private fun lagOpptegnelse() {
         val opptegnelseType: OpptegnelseType = when {
-            type == Utbetalingtype.ANNULLERING && status == UTBETALING_FEILET -> {
-                OpptegnelseType.UTBETALING_ANNULLERING_FEILET
-            }
             type == Utbetalingtype.ANNULLERING && status == ANNULLERT -> {
                 OpptegnelseType.UTBETALING_ANNULLERING_OK
             }
-            type == Utbetalingtype.REVURDERING && status in listOf(UTBETALT, GODKJENT_UTEN_UTBETALING, OVERFØRT) -> {
+            type == Utbetalingtype.REVURDERING && status in listOf(UTBETALT, GODKJENT_UTEN_UTBETALING) -> {
```

Comment:
Finnes det noen erstatning her, eller tilsvarende? Tidligere har vi akseptert `OVERFØRT` som god nok status for å si at en revurdering er ferdigbehandlet.

Reply:
aha, did not know.

Ja: `AVVENTER_KVITTERINGER, AVVENTER_ARBEIDSGIVERKVITTERING, AVVENTER_PERSONKVITTERING`